Astro CLI v1.0.0 (All Changes) (#690)
astronomer/docs: software/deploy-nfs.md

Starting in Astronomer Software v0.25, you can use an external [Network File System (NFS) Volume](https://kubernetes.io/docs/concepts/storage/volumes/#nfs) to deploy DAGs to an Airflow Deployment on Astronomer.

Unlike [deploying DAGs via the Astro CLI](deploy-cli.md), deploying DAGs to an NFS volume, such as Azure File Storage or Google Cloud Filestore, does not require rebuilding a Docker image and restarting your underlying Airflow service. When a DAG is added to an NFS volume, it automatically appears in the Airflow UI without requiring additional action or causing downtime.

This guide provides the necessary setup for connecting an NFS volume to Astronomer as a DAG deploy option. Some considerations before completing this setup:

> - You can configure NFS volumes only to deploy DAGs. To push dependencies or other requirements to your Airflow Deployment, you still need to update your `requirements.txt` and `packages.txt` files and deploy via either the [Astro CLI](deploy-cli.md) or [CI/CD](ci-cd.md). For more information on pushing code to your Airflow environment, read [Customize Images](customize-image.md).

Reviewer: ```suggestion
- You can configure NFS volumes only to deploy DAGs. To push dependencies or other requirements to your Airflow Deployment, you'll need to update your `requirements.txt` and `packages.txt` files and deploy using the [Astro CLI](deploy-cli.md) or [CI/CD](ci-cd.md). For more information on pushing code to your Airflow environment, see [Customize Images](customize-image.md).
```

> - If you configure an NFS volume for an Airflow Deployment, you can't deploy DAGs via the Astro CLI or an Astronomer service account. These options are available only for Deployments configured with an image-based deploy mechanism.

Reviewer: ```suggestion
- If you configure an NFS volume for an Airflow Deployment, you can't use the Astro CLI or an Astronomer service account to deploy DAGs . These options are available only for Deployments configured with an image-based deploy mechanism.
```

4. In the **NFS Location** field that appears, enter the location of your volume-based DAG directory as `<IP>:/<path>` (for example: `192.168.0.1:/path/to/your/dags`).
5. Save your changes.

> > **Note:** NFS volumes can also be configured via the Astro CLI. To do so, specify the `--nfs-location` flag when running [`astro deployment create`](cli-reference.md#astro-deployment-create) or [`astro deployment update`](cli-reference.md#astro-deployment-update).

Reviewer: ```suggestion
> **Note:** You can also use the  Astro CLI to configure NFS volumes. To do so, specify the `--nfs-location` flag when running [`astro deployment create`](cli-reference.md#astro-deployment-create) or [`astro deployment update`](cli-reference.md#astro-deployment-update).
```
